kfctl/README.md

```diff
@@ -13,14 +13,14 @@ be implemented in golang. The port to golang is because:
 
 The initial version of kfctl will seek parity with kfctl.sh by implementing the following subcommands:
 - `init`            Initialize a kubeflow application.
-- `generate`        Generate the k8 manifests of the kubeflow application.
-- `apply`           Submit the k8 manifests to the api-server
+- `generate`        Generate the k8s manifests of the kubeflow application.
+- `apply`           Submit the k8s manifests to the api-server
 - `delete`          Delete the kubeflow application
 
 Typical usage of `kfctl.sh` is as follows:
```

Comment:
I think we deleted kfcth.sh so can you delete all the related instructions.

Reply:
@jlewi Thanks for your review, I will close this PR and create a new one to delete kfcth.sh related instructions.